assets/queries/dockerfile/run_utilities_and_posix_commands/query.rego

```diff
@@ -16,18 +15,5 @@ CxPolicy[result] {
 }
 
 containsCommand(cmds) {
-	commands = [
-		"shutdown",
-		"service",
-		"ps",
-		"free",
-		"top",
-		"kill",
-		"mount",
-		"ifconfig",
-		"nano",
-		"vim",
-	]
-
-	contains(cmds.Value[_], commands[_])
+	regex.match("\\b( *ps *)\\b|\\b( *shutdown *)\\b|\\b( *service *)\\b|\\b( *free *)\\b|\\b( *top *)\\b|\\b( *kill *)\\b|\\b( *mount *)\\b|\\b( *ifconfig *)\\b|\\b( *nano *)\\b|\\b( *vim *)\\b", cmds.Value[_])
```

Comment:
the use of this regex serves to fix what purpose? the fact that it may contain more spaces between the commands?

Reply:
It is for spaces and to ensure that words like 'ps' are not captured in 'https: ....', or other examples